src/getting-started/starter-templates.md

```diff
@@ -22,7 +22,7 @@ crate-type = ["cdylib"]
 
 [dependencies]
 # for web_sys
-yew = 0.15
+yew = "0.15"
```

Comment:
Would you mind bumping up the version to the latest version of Yew?
```suggestion
yew = "0.16"
```

Reply:
Nope

Reply:
Thanks (I assume you mean no to not minding, rather than no to updating the version)!

Reply:
Yes right